docs/interactions/Slash_Commands.md

```diff
@@ -891,6 +891,7 @@ An application command is the base "command" model that belongs to an applicatio
 | description         | string                                                                                                                                   | 1-100 character description                                                |
 | options?            | array of [application command option](#DOCS_INTERACTIONS_SLASH_COMMANDS/application-command-object-application-command-option-structure) | the parameters for the command                                             |
 | default_permission? | boolean (default `true`)                                                                                                                 | whether the command is enabled by default when the app is added to a guild |
+| version             | snowflake                                                                                                                                | snowflake of last time the command was edited                              |
```

Comment:
```suggestion
| version             | snowflake                                                                                                                                | last modification time                                                            |
```

Reply:
A snowflake is not a time so this description is misleading.

Reply:
tbh i don't think this is any better

Reply:
yeah I don't know myself but if it should be documented than it should be named as modification xyz. 
A whatever